eng/pipelines/coreclr/exploratory.yml

```diff
@@ -0,0 +1,63 @@
+# This job definition automates the exploratory tool.
```

Comment:
Should all these be named specifically for the JIT, so it's clear that it's only the JIT being tested? E.g., name this `jit-exploratory.yml`, etc.?

Reply:
Sure.